python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -243,14 +248,18 @@ def _getDefaultValueInternal(self, attr, val=None, check=False):
                 defvalue = _found_attrs[stored_attr_key]
             else:
                 defvalue = config[attr]
-                _found_attrs[stored_attr_key] = defvalue
+                from Ganga.GPIDev.Base.Proxy import stripProxy
+                ## Just in case a developer puts the proxied object into the default value!
+                _found_attrs[stored_attr_key] = stripProxy(defvalue)
```

Comment:
I'm happy to rather test for isProxy and raise an error but I was hoping to avoid this particular problem for now in favour of getting things working.
If we want to take a stance on it, it likely would have turned into more work I have to do on this branch.